src/status_im2/contexts/communities/menus/channel_options/view.cljs

```diff
@@ -0,0 +1,110 @@
+(ns status-im2.contexts.communities.menus.channel-options.view
+  (:require [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [quo2.core :as quo]))
+
+(defn hide-sheet-and-dispatch
+  [event]
+  (rf/dispatch [:bottom-sheet/hide])
+  (rf/dispatch event))
+
+(defn mute-channel-action
+  [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/mute chat-id true]))
+
+(defn unmute-channel-action
+  [chat-id]
+  (hide-sheet-and-dispatch [:chat.ui/mute chat-id false]))
+
+(defn view-members
+  [id]
+  {:icon                :i/members
+   :accessibility-label :view-members
+   :label               (i18n/label :t/view-channel-members-and-details)})
+
+(defn view-token-gating
+  [id]
+  {:icon                :i/token
+   :right-icon          :i/chevron-right
+   :chevron?            true
+   :accessibility-label :view-token-gating
+   :on-press            #(js/alert (str "implement action" id))
+   :label               (i18n/label :t/view-token-gating)})
+
+(defn mark-as-read
+  [id]
+  {:icon                :i/up-to-date
+   :accessibility-label :mark-as-read
+   :label               (i18n/label :t/mark-as-read)
+   :on-press            #()})
+
+(defn mute-channel
+  [id muted?]
+  {:icon                (if muted? :i/muted :i/activity-center)
+   :icon-color          :red
```

Comment:
colors must be from color ns

Reply:
Sorry, Was a testing value and forgot to remove